internal/explorer.point/package.json

```diff
@@ -28,7 +28,7 @@
     "wouter": "^2.7.4"
   },
   "scripts": {
-    "clear": "mkdir -p \"./public\" && rm -rf \"./public/*\" \".cache\" \".parcel-cache\"",
```

Comment:
I assume you had to remove the `mkdir` call because it doesn't work on Windows, but unfortunately it breaks the tests, please see the error [here](https://github.com/pointnetwork/pointnetwork/runs/8112509486?check_suite_focus=true).

One solution could be to ignore errors from `rm` (like sending stderr to /dev/null), but not sure if that kind of redirection would work on Windows. Another solution could be to use a cross-platform tool for `mkdir -p`.

Reply:
Actually, given the `build` script, in explorer.point, we need the `public/` directory to be created. So, unless we modify `build`, my first alternative solution above won't work.